apps/tailwind-components/components/Modal.vue

```diff
@@ -78,12 +78,12 @@ function hide() {
     />
 
     <div
-      class="bg-modal w-3/4 relative m-auto rounded-t-none rounded-b-theme"
+      class="bg-modal w-3/4 relative m-auto rounded-t-none rounded-b-theme h-[95vh] flex flex-col"
       :class="maxWidth"
     >
       <slot name="header">
         <header
-          class="pt-[36px] px-[30px] overflow-y-auto border-b border-divider"
+          class="pt-[36px] px-[30px] flex-none overflow-y-auto border-b border-divider"
         >
           <div v-if="subtitle" class="text-title-contrast">{{ subtitle }}</div>
```

Comment:
More of a note for later: Title and subtitles should be marked up as a heading elements. Titles should have the highest level and subtitle should be the next level. Depending on the context of the modal, the heading levels should be h2 and h3.